firmware/user/display/font.h

```diff
@@ -12,6 +12,10 @@
 #include "sprite.h"
 #include "oled.h"
 
+// Ref: http://marrin.org/2017/01/16/putting-data-in-esp8266-flash-memory/
```

Comment:
This would be more useful than just for fonts

Reply:
Richard did this, but we discussed that handling of the other data in rom, needsto handled similarly